daily_read/ngi_data.py

```diff
@@ -185,7 +189,16 @@ class ProjectDataRecord(object):
     Raises ValueError if orderer is not present in data, if data is given
     """
 
-    def __init__(self, relative_path, data=None):
+    dates_prio = {
```

Comment:
We should probably have a single source of the priority, could modify `STATUS_PRIORITY` from main. Something to do in the future.

Reply:
Created https://github.com/NationalGenomicsInfrastructure/DailyRead/issues/21 to not completely forget this.